api/swagger/swagger-dev.json

```diff
@@ -21438,6 +21438,150 @@
         }
       }
     },
+    "/v1/management/backup/Backups/List": {
```

Comment:
The url should be consistent with the rest of the management APIs, PTAL https://pmmdemo.percona.com/swagger#/